chapters/statements.tex

```diff
@@ -289,20 +285,19 @@ \subsubsection{Types as Iteration Ranges }\doublelabel{types-as-iteration-ranges
     xsquared1[i]=xe[i]^2;
   end for;
 \end{lstlisting}
-{]}
+\end{example}
 
 \subsubsection{Nested For-Loops and Reduction Expressions with Multiple Iterators}\doublelabel{nested-for-loops-and-reduction-expressions-with-multiple-iterators}
 
 The notation with several iterators is a shorthand notation for nested
 for-statements or for-equations (or reduction-expressions). For
 for-statements or for-equations it can be expanded into the usual form
-by replacing each ``,'' by `\lstinline!loop for!' and adding extra `\lstinline!end for!'. For
+by replacing each `\lstinline!,!' by \lstinline!loop for! and adding extra \lstinline!end for!. For
```

Comment:
The quoting here seems inconsistent with the one a few lines below (after changes). It was inconsistent before as well, but almost reverse.

Reply:
OK.  I tried to fix it by consistently using double quotes around the incomplete fragments of inline code.  They are added to clarify the start and end of the incomplete fragments of code, to avoid attempts at parsing a bigger chunk of text to end up with something less incomplete (an expression, for instance).  Would you prefer to not use double quotes for this purpose?

Reply:
Either double quotes work or no quotes work; provided long as line 295 ("by replacing each") and 297("replacing each") use them consistently.

Reply:
OK, then I'll leave it as it is now.